openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -310,15 +308,15 @@ def test_home_page(
                 private_url = course_home_url(self.private_course)
                 private_response = self.client.get(private_url)
 
-        # Verify that the course tools and dates are always shown
```

Comment:
For anonymous or unenrolled users, they previously only saw the Course Reviews course tool. Without this tool, only logged in/staff users now see any course tools.